pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -465,18 +465,8 @@ func (cm *containerManagerImpl) setupNode(activePods ActivePodsFunc) error {
 
 	if cm.KubeletCgroupsName != "" {
 		cont := newSystemCgroups(cm.KubeletCgroupsName)
-		allowAllDevices := true
-		manager := fs.Manager{
-			Cgroups: &configs.Cgroup{
-				Parent: "/",
-				Name:   cm.KubeletCgroupsName,
-				Resources: &configs.Resources{
-					AllowAllDevices: &allowAllDevices,
-				},
-			},
-		}
-		cont.ensureStateFunc = func(_ *fs.Manager) error {
-			return ensureProcessInContainerWithOOMScore(os.Getpid(), qos.KubeletOOMScoreAdj, &manager)
+		cont.ensureStateFunc = func(manager *fs.Manager) error {
```

Comment:
From looking at https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/cm/container_manager_linux.go#L338, I believe you are correct that this `fs.Manager` is identical to the other `cont.Manager` which we already created.